api/nodes/serializers.py

```diff
@@ -110,6 +118,7 @@ def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
     def create(self, validated_data):
+        validated_data.pop('type')
```

Comment:
Isn't type already pretty special in DRF, or do you want it to be special in a different way?